AirCasting/SessionViews/SessionLoadingView.swift

```diff
@@ -0,0 +1,29 @@
+// Created by Lunar on 24/06/2021.
+//
+
+import SwiftUI
+
+struct SessionLoadingView: View {
+    var body: some View {
+        HStack {
+            Image("ABLoading")
+                .frame(width: 60, height: 60)
```

Comment:
Is this working well all the time? Wondering 💭 because .resizable modifier is not here